index.d.ts

```diff
@@ -907,7 +907,7 @@ type AppendKeyPrefix<Key, KPrefix> = KPrefix extends string
  * T function declaration *
  **************************/
 export interface TFunction<Ns extends Namespace = _DefaultNamespace, KPrefix = undefined> {
-  $TFunctionBrand: Ns;
+  $TFunctionBrand: Ns extends string ? `${Ns}` : never;
```

Comment:
I don't really understand why this `` `${Ns}` `` is necessary, but it doesn't work if I just use `Ns` 😕